testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -472,26 +472,26 @@ public void testShowCreateTable()
     public void testView()
     {
         if (!hasBehavior(SUPPORTS_CREATE_VIEW)) {
-            assertQueryFails("CREATE VIEW nation_v AS SELECT * FROM nation", "This connector does not support creating views");
```

Comment:
`nation` table is quite small, 25 rows, do we need to change this ?

Reply:
`testView` and `testView` take 1m and 2m respectively for a certain connector, but again these are not the slowest ones by far, so we can roll these back if you think it is important to have a (slightly) bigger table here.

Reply:
The change is okay, we don't care about size of table here. I'm just surprised that a difference of 5 vs 25 rows matters.

Reply:
Totally agree, but for some source systems writes are extremely slow (especially in CI situation where we use smaller instances). As it is not critical, I will roll back these 2.